src/inference/enumerate.ad.js

```diff
@@ -17,8 +17,18 @@ module.exports = function(env) {
   function Enumerate(store, k, a, wpplFn, options) {
     util.throwUnlessOpts(options, 'Enumerate');
     options = util.mergeDefaults(options, {
-      maxExecutions: Infinity
+      maxExecutions: Infinity,
+      throwOnError: true,
+      maxRuntimeInMS: Infinity,
+      maxEnumTreeSize: Infinity
```

Comment:
It's a little weird that `maxExecutions` is treated differently from `maxRuntimeInMS` and `maxEnumTreeSize`. (The former returns a partial distribution, the latter error objects.) We might want to unify their treatment in the future (but probably not in this PR).